scripts/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,538 @@
+// FIXME sloc publish errors
+// XXX add missing jobs
+// TODO have a per plugin/binding deps in Dockerfile for easier maintanence
+// TODO add warnings plugins to scan for compiler warnings
+
+// # Imports #
+import java.text.SimpleDateFormat
+
+// # Set Variables #
+CMPVERSION = '0.8.22'
+VERSION = '0.8.23'
```

Comment:
doc/todo/RELEASE should have information when this needs to be updated.

Reply:
since it is currently not used and was only there as an example i can simply remove it

Reply:
For the icheck and the Debian jobs you will need them. But for this PR you can remove them if you prefer so.

Reply:
Hopefully we can parse the version numbers from somewhere in the repository?

Reply:
Why parse? The current version number is in the CMake variable KDB_VERSION.

But the numbers above are not the current version but the version to build (Debian Packages for) and the version to compare to. They differ from KDB_VERSION respective KDB_VERSION-1 during the release process.